src/main/java/jenkins/scm/api/SCMNavigatorDescriptor.java

```diff
@@ -41,6 +42,46 @@ protected SCMNavigatorDescriptor(Class<? extends SCMNavigator> clazz) {
     }
 
     /**
+     * A description of this kind of {@link SCMNavigator}. This description can contain HTML code but it is recommend
+     * to use text plain in order to avoid how it should be represented.
+     *
+     * @return A string, by default the value from newInstanceDetail view is taken.
+     *
+     * @since 1.2
+     */
+    @Nonnull
+    public String getDescription() {
+        return Messages.SCMNavigator_Description();
+    }
+
+    /**
+     * Used to categorize this kind of {@link SCMNavigator}.
+     *
+     * @return A string with the category identifier.
```

Comment:
Obviously you cannot use `@link`, but you _can_ use `@code` and leave a TODO comment to hyperlink it later.